bookwyrm/views/rss_feed.py

```diff
@@ -0,0 +1,29 @@
+''' '''
+
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from bookwyrm.models.user import User
+from .helpers import get_activity_feed, get_user_from_username
+
+class RssFeed(Feed):
+
+    description_template = "snippets/rss_content.html"
+    title_template = "snippets/rss_title.html"
+
+    def get_object(self, request, username):
+        return get_user_from_username(username)
+
+    def link(self, obj):
+        return obj.local_path
+
+    def title(self, obj):
+        return f"Status updates from {obj.display_name}"
+
+
+    def items(self, obj):
+        return get_activity_feed(obj, ['public', 'unlisted'])
```

Comment:
okay sorry I just realized this, and it's entirely a result of the get_activity_feed function being super unclear! the way you're calling it right now gets the activity feed for the current user (`obj`), which means posts from users they follow as well as their own. 

in order to limit it to just the posts by the user, you have to pass in a queryset. which is weird and confusing! sorry. if you call `get_activity_feed(obj, ['public', 'unlisted'], queryset=obj.status_set)` it should work correctly